cabby/geo/osm.py

```diff
@@ -0,0 +1,57 @@
+# coding=utf-8
+# Copyright 2020 Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+'''Defines OSM entities adaptations.'''
+
+CORRECTIONS = {'fuel': 'gas station', 'parking': 'parking lot', 'taxi': 'taxi stand', 'doctors': 'doctor\'s office', 'dentist': 'dentist\'s office', 'gambling': 'gambling place', 'police': 'police station', 'shower': 'public bath', 'mall shop': 'mall', 'general shop': 'shop', 'supermarket shop': 'supermarket', 'toys shop': 'toy shop', 'fast food': 'fast food restaurant'}
```

Comment:
Please change this to one item per line. Also, are the gambling places 'casinos', or does it possibly other betting establishments?